src/main/resources/application.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+  datasource:
+    url: jdbc:h2:mem:mydb
```

Comment:
are you sure with this settings ? I think you configured it as JDBC

Reply:
fixed.